ledger/store/trackerdb/sqlitedriver/accountsV2.go

```diff
@@ -365,7 +367,11 @@ func (r *accountsV2Reader) LoadTxTail(ctx context.Context, dbRound basics.Round)
 }
 
 // LookupAccountAddressFromAddressID looks up an account based on a rowid
-func (r *accountsV2Reader) LookupAccountAddressFromAddressID(ctx context.Context, addrid int64) (address basics.Address, err error) {
+func (r *accountsV2Reader) LookupAccountAddressFromAddressID(ctx context.Context, accountRef trackerdb.AccountRef) (address basics.Address, err error) {
+	if accountRef == nil {
+		return address, sql.ErrNoRows
```

Comment:
`err = fmt.Errorf("no matching address could be found for rowid %d: %w", addrid, err)` as below?